### ISSUE TITLE
ADD EXPIRY DATE OF BATCHES DURING PURCHASING

### ISSUE THREAD

**Issue by ahmedbahgattt** [frappe/erpnext#32321]
THE PROBLEM 
 in any retail business we need to add expiry date of items 
(hundreeds)  while adding purchase recipiant 
for tracking real stock of expiry items and many reasons

SOLUTION
the simpliest way is  adding batch expiry date in 
purchase recipuant item   (child table)   not only batch 

and this us manadatory and critical to any retail which dealing with thousands of items

**Comment by f-9t9it** [comment]
We already have added this as a customization. It is not preferable for most of the businesses hence not in core.

**Comment by pmjd-code** [comment]
I agree with you @ahmedbahgattt , this is a missing feature which should be in the core as an option. Having to retrospectively add Manufacturer/Supplier's expiry dates to items after receipt is time consuming, when it could more easily and effectively be done when items are added on a Purchase Receipt.
